lib/navigation/imu_displacement.cpp

```diff
@@ -0,0 +1,44 @@
+#include "imu_displacement.hpp"
+
+#include <cmath>
+
+#include <chrono>
```

Comment:
does clang-format group the headers like this? i'd be surprised if it adds a space between these two; if that's the case we should fix the `.clang_format` to not do that

Reply:
It does. I'll have a look into it later.